server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/Attribute.java

```diff
@@ -67,6 +67,9 @@
     DATASOURCE(ModelKeys.DATASOURCE),
     DEFAULT_CACHE(ModelKeys.DEFAULT_CACHE),
     @Deprecated DEFAULT_CACHE_CONTAINER("default-cache-container"),
+    DB_DISABLE_UPSERT(ModelKeys.DB_DISABLE_UPSERT),
```

Comment:
Not sure I like having a dedicated configuration option for this.
Better have it as a "known" property (to be specified with the <properties> element).